metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -39,29 +39,25 @@ class KubernetesDecorator(StepDecorator):
 
     Parameters
     ----------
-    cpu : int
-        Number of CPUs required for this step. Defaults to 1. If `@resources` is
+    cpu : int, default: 1
+        Number of CPUs required for this step. If `@resources` is
         also present, the maximum value from all decorators is used.
-    memory : int
-        Memory size (in MB) required for this step. Defaults to 4096 (4GB). If
+    memory : int, default: 4096
+        Memory size (in MB) required for this step. If
         `@resources` is also present, the maximum value from all decorators is
         used.
-    disk : int
-        Disk size (in MB) required for this step. Defaults to 10GB. If
+    disk : int, default: 10240
+        Disk size (in MB) required for this step. If
         `@resources` is also present, the maximum value from all decorators is
         used.
-    image : str
+    image : str, optional, default: None
```

Comment:
this default isn't wholly accurate

Reply:
I put it as optional and described tthe behavior in the doc string.

Reply:
the behavior is symmetric to batch - so the comment should be quite similar here

Reply:
stupid me -- I had modified it but forgot to save the file.

Reply:
stupid me -- I had modified it but forgot to save the file.